x/service/types/genesis_test.go

```diff
@@ -6,9 +6,20 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/pokt-network/poktroll/x/service/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
 )
 
 func TestGenesisState_Validate(t *testing.T) {
+	srv1 := &sharedtypes.Service{
+		Id:   "srv1",
+		Name: "srv1",
+	}
+
+	srv2 := &sharedtypes.Service{
```

Comment:
ULTRA NIT (not going to ask for it but sharing): In the past, I've seen `svc` as an acronym for `Service` and `srv` as an acronym for `Server`. Only make these changes if you're really feeling up to it.